tools/feature_version_audit.py

```
@@ -0,0 +1,685 @@
+        missing_fields = []
+        if not info['is_core'] and not info['mod_link']:
+            missing = True
+            missing_fields.append('mod link')
+        if not info['description']:
+            missing = True
+            missing_fields.append('description')
+        if not info['key_features']:
+            missing = True
+            missing_fields.append('key features')
+        def boldmeta(val):
+            return f"**{val}**" if missing else val
```

Ruff (0.11.9)

Function definition does not bind loop variable `missing`

(B023)